src/trivial_demonstrations/sum_of_squares.rs

```diff
@@ -0,0 +1,9 @@
+use std::io;
+fn main() {
+    let n:i32;
```

Comment:
remove this line. its defined again when assigning

program works with warning

Reply:
No. There is no shortcut.